Website/plugins/sqlite-db/compilation-callable.raku

```diff
@@ -0,0 +1,27 @@
+use v6.d;
+sub ( $pr, %processed, %options) {
+    my $sql;
+    my %config = $pr.get-data('sqlite-db');
+    if $pr.plugin-datakeys (cont) 'tablemanager' {
+        my @rows = $pr.get-data('tablemanager').<dataset><routines>.list;
+        $sql = q:to/SQL/;
+            INSERT INTO routines ( Category, Name, Type, URL )
+            VALUES
+            SQL
+        $sql ~= [~] @rows[0 ^..* ].map({
+                '("' ~ .[0] ~ '" , "' ~ .[1] ~ '" , "'~ .[2] ~ '" , "'
+               ~ .[3] ~ '#' ~ .[4]
+               ~ '")'
+               })
+               .join(",\n") ~ ";\n";
+    }
+    else {
+        # change the line below to create a string that will cause a better sqlite result
+       $sql = 'There is no tablemanager data'
+    }
+    %config<db-filename>.IO.spurt: $sql;
+    [
+        [ %config<database-dir> ~ '/schema.sql' , 'myself', 'schema.sql' ],
+        [ %config<database-dir> ~ '/' ~ %config<db-filename> , 'myself', %config<db-filename> ]
```

Comment:
What does 'myself' mean in this context?

Reply:
'myself' means from the plugin's own directory, as opposed to another plugin directory. By the time I had written a lot of plugins, and realised this extra generality was not needed, I had no time to refactor.
The aim is for the Collection instance to handle transfers of files bundled or created in a plugin directory to where the files will be served.